platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformComponentBuilder.java

```diff
@@ -403,4 +415,53 @@ public RunningEventHasher buildRunningEventHasher() {
         }
         return runningEventHasher;
     }
+
+    /**
+     * Provide an event creation manager in place of the platform's default event creation manager.
+     *
+     * @param eventCreationManager the event creation manager to use
+     * @return this builder
+     */
+    @NonNull
+    public PlatformComponentBuilder withEventCreationManager(@NonNull final EventCreationManager eventCreationManager) {
```

Comment:
I recall you mentioned there may be legitimate need for some of these unused methods but i suggest at the very least, they should be called by some unit tests that asserts their functionality.

Reply:
At the time of our previous conversation these methods were not called, but that is not true any more. They are currently called in `PlatformWiringTests.java`.

![Screenshot 2024-04-12 at 11 34 14 AM](https://github.com/hashgraph/hedera-services/assets/56973212/c7243449-4d0f-4771-a6f6-2fc49e7569f2)